masschallenge/django-accelerator
# AC-8644 :: P1: Expert Overview displays to finalists

**Changes introduce in [AC-8644](https://masschallenge.atlassian.net/browse/AC-8644)**
- added a user_type flag to the UserRoleMenu
- filtered associations when user_type is specified

**How to test**
- On test4, as an admin, go to the expert user role menu. 
- You will see a selection box for user_type. 
- Select Expert and save
- You can then masquerade as an entrepreneur and see that you no longer see the `Expert Overview`
- Masquerade as an expert and see that you see the `Expert Overview`
